source/gfx.c

```diff
@@ -226,6 +226,8 @@ bool S9xInitGFX(void)
    GFX.PPLx2 = GFX.Pitch;
    S9xFixColourBrightness();
 
+#if defined(PSP)
```

Comment:
I think a better approach would be to have a new flag (something like `OLD_COLOUR_OPS`) and defining it by default only for PSP target. This way, we can disable this in some specific target in case this code causes some problem (it shouldn't, since I tested this for quite sometime in https://github.com/m45t3r/snes9x4d-miyoo/, but you never know).

But I am not familiar with libretro code, if using specific targets is the way preferred here :+1: .

Reply:
Thanks for the feedback!

Yes, I like the `OLD_COLOUR_OPS` flag idea. I'll change this.